app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewsController < ApplicationController
+  def create
+    #is this the proper way to pass in params?
+
+    new_review_data = review_params
+    new_review_data[:user_id] = current_user.id
```

Comment:
And for setting the user ID, instead of adding it to `review_params`, after line 8 you can add a line that says something like `@vendor.user = current_user`